API/Parser/Parser.cs

```diff
@@ -136,213 +136,121 @@ public static class Parser
         {
             // Grand Blue Dreaming - SP02
             new Regex(
-                @"(?<Series>.*)(\b|_|-|\s)(?:sp)\d",
+                $@"{seriesExpr}(\b|-|\s)(?:sp)\d",
                 MatchOptions, RegexTimeout),
-            // [SugoiSugoi]_NEEDLESS_Vol.2_-_Disk_The_Informant_5_[ENG].rar, Yuusha Ga Shinda! - Vol.tbd Chapter 27.001 V2 Infection ①.cbz
+            // Yuusha Ga Shinda! - Vol.tbd Chapter 27.001 V2 Infection ①.cbz
             new Regex(
-                @"^(?<Series>.*)( |_)Vol\.?(\d+|tbd)",
+                $@"^{seriesExpr} Vol\.?tbd\b",
                 MatchOptions, RegexTimeout),
+            // [SugoiSugoi] NEEDLESS Vol.2 - Disk The Informant 5 [ENG].rar
             // Mad Chimera World - Volume 005 - Chapter 026.cbz (couldn't figure out how to get Volume negative lookaround working on below regex),
             // The Duke of Death and His Black Maid - Vol. 04 Ch. 054.5 - V4 Omake
-            new Regex(
-                @"(?<Series>.+?)(\s|_|-)+(?:Vol(ume|\.)?(\s|_|-)+\d+)(\s|_|-)+(?:(Ch|Chapter|Ch)\.?)(\s|_|-)+(?<Chapter>\d+)",
-                MatchOptions,
-                RegexTimeout),
-            // Ichiban_Ushiro_no_Daimaou_v04_ch34_[VISCANS].zip, VanDread-v01-c01.zip
```

Comment:
Same here, multiple cases removed, I don't see a comment for their case. Please ensure there is a unit test matching all these cases as well.

Reply:
Especially since I don't see any MangaParserTests in this PR.

Reply:
I have not added new patterns to the manga regex, hence why there's no addition to MangaParserTests.
Wasn't the coverage good enough before?

Reply:
The cases matching regex that have been removed are matched by other regex that have become more generic. I have moved the the comments describing use cases to the new regex in order to track where they are being matched now.